include/deal.II/base/mpi_noncontiguous_partitioner.templates.h

```diff
@@ -54,12 +54,19 @@ namespace Utilities
 
 
     std::pair<unsigned int, unsigned int>
-    NoncontiguousPartitioner::n_targets()
+    NoncontiguousPartitioner::n_targets() const
     {
       return {send_ranks.size(), recv_ranks.size()};
     }
 
 
+    unsigned int
```

Comment:
Can we add a third empty line above this?

Reply:
Done.